### PR TITLE
Fix spelling + indentation

### PR DESCRIPTION
**Pull request by RincewindsHat**

### CODE REVIEW

**Comment by RincewindsHat** [comment]
ok, that escalated quickly.

File: result/overall.go [hunk 1]
```diff
@@ -4,13 +4,16 @@ package result
...
 	"fmt"
+	"strconv"
 	"strings"
 
 	"github.com/NETWAYS/go-check"
 	"github.com/NETWAYS/go-check/perfdata"
 )
 
-// Overall is a singleton for a monitoring pluging that has several partial results (or sub-results)
+const indentationOffset = 2
```
Comment (martialblog) [inline result/overall.go:14]: Needs a comment with a description

**Review by martialblog — COMMENTED** [review]
Looks alright to me

**Review by martialblog — APPROVED** [review]

**Comment by martialblog** [comment]
shipit!

**Pull request merged**